app/src/js/components/worker_dashboard.tsx

```diff
@@ -132,7 +136,9 @@ function Dashboard(props) {
 /** **************/
 // post Ajax request for projects
 let xhrproj = new XMLHttpRequest();
-let projectsToexpress;
+let projectsToexpress: {
```

Comment:
nit: slightly easier to read

```suggestion
let projectsToExpress: {
```